packages/clerk-js/src/core/resources/OrganizationMembership.ts

```diff
@@ -18,6 +19,8 @@ export class OrganizationMembership extends BaseResource implements Organization
   publicMetadata: OrganizationMembershipPublicMetadata = {};
   publicUserData!: PublicUserData;
   organization!: Organization;
+  // eslint-disable-next-line
```

Comment:
Note: adding `(string & {})` allows for getting eslint autocomplete but also accepts any string

Reply:
good call